providers/fips/self_test_data.inc

```diff
@@ -494,8 +494,76 @@ static const ST_KAT_PARAM kbkdf_params[] = {
     ST_KAT_PARAM_END()
 };
 
+#ifndef OPENSSL_NO_TLS1_3
+static const char tls13_kdf_digest[] = "SHA256";
+static int tls13_kdf_extract_mode = 1;
+static int tls13_kdf_expand_mode = 2;
```

Comment:
Why the magic numbers and not the constants?

Reply:
Good point.
Will change.